package.json

```diff
@@ -2,7 +2,7 @@
   "name": "tck",
   "description": "AsyncAPI test compatibility kit",
   "version": "1.1.0",
-  "main": "genmanifest.js tests/asyncapi-2.0",
+  "main": "",
```

Comment:
If there's no "main", you can remove it.

Reply:
Removed.